cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -691,15 +806,15 @@ arrow::Status VeloxShuffleWriter::splitFixedWidthValueBuffer(const velox::RowVec
           multiply = std::min(3, multiply + 1);
 
           auto valueBuffer = std::static_pointer_cast<arrow::ResizableBuffer>(
-              partitionBuffers_[fixedWidthColumnCount_ + binaryIdx][pid][kValueBufferInedx]);
+              partitionBuffers_[fixedWidthColumnCount_ + binaryIdx][pid][kValueBufferIndex]);
 
           RETURN_NOT_OK(valueBuffer->Reserve(capacity));
 
-          binaryBuf.value_ptr = valueBuffer->mutable_data();
-          binaryBuf.value_capacity = capacity;
-          dstValuePtr = binaryBuf.value_ptr + valueOffset - stringLen;
+          binaryBuf.valuePtr = valueBuffer->mutable_data();
+          binaryBuf.valueCapacity = capacity;
+          dstValuePtr = binaryBuf.valuePtr + valueOffset - stringLen;
 
-          VsPrintSplit("Split value buffer resized colIdx", binaryIdx);
+          std::cout << "Split value buffer resized colIdx" << binaryIdx << std::endl;
```

Comment:
@rui-mo Should we remove this log?

Reply:
Maybe change to only dump the log when debugging?